README.md

```diff
@@ -4,15 +4,13 @@ These packages provide utilities for generating Dart code.
 
 ## Build
```

Comment:
Use `build` on the header – so it's clear you're talking about the subdirectory/package